frontend/src/views/Join.tsx

```diff
@@ -73,6 +73,8 @@ function JoinGamePage() {
       getRoom(roomIdParam)
         .then((res) => {
           setLoading(false);
+
+          // TODO: Check to be removed to allow spectator view.
```

Comment:
Is this TODO still relevant?

Reply:
Yes it is; for the next PR, this will need to be removed so people who are joining active games are automatically created as spectators, and are thus actually able to join the active game.